libsolidity/interface/CompilerStack.h

```diff
@@ -290,7 +290,8 @@ class CompilerStack: boost::noncopyable
 		std::string target;
 	};
 
-	ReadFile::Callback m_readFile;
+	ReadCallback::Callback m_readFile;
+	ReadCallback::Callback m_smtQuery;
```

Comment:
Yeah, let's keep it like that for now. This is the endpoint where systems that do not have libz3 tie in, for example the jsonCompiler. We still have to decide how the interface for specifying such a callback would look like.